gelfformatter/formatter.py

```diff
@@ -89,12 +89,17 @@ class GelfFormatter(logging.Formatter):
        https://docs.python.org/3/library/logging.html#logrecord-attributes
     """
 
-    def __init__(self, allowed_reserved_attrs=[]):
+    def __init__(self, allowed_reserved_attrs=[], additional_reserved_attrs=[]):
```

Comment:
We should pick a different name for this new argument, `.*reserved_attrs` is better suited for the standard library attributes (i.e. "reserved" means defined/owned somewhere else). 

Something shorter like `ignored_attrs` would be more accurate and clean. In the end, we're adding functionality to enable us to ignore attributes, regardless if they're reserved (from the std lib) or custom (from our apps).